src/NServiceBus.Testing.Fakes/TestableBehaviorContext.cs

```diff
@@ -40,6 +41,7 @@ protected virtual IServiceProvider GetBuilder()
             return builder;
         }
 
+        [SuppressMessage("Code", "PCR0003:Instance methods on types implementing ICancellableContext should not have a CancellationToken parameter", Justification = "Designed for testing.")]
```

Comment:
Instance methods? Is the analyzer looking at `set_CancellationToken` as a method?

I could see just exempting this entire project.

Reply:
> Instance methods? Is the analyzer looking at `set_CancellationToken` as a method?

It is. I thought about refining the analyzer to do something special with property setters, but it kind of makes sense as-is, and this is a one-off violation.

> I could see just exempting this entire project.

I'd prefer to be specific first, and more general only if we have to be. Ideally, I think we want any testing libs to have proper token support?